examples/plots/Station_Plot.py

```diff
@@ -85,25 +69,29 @@
 
 # Map weather strings to WMO codes, which we can use to convert to symbols
 # Only use the first symbol if there are multiple
-wx_text = [s.decode('ascii') for s in data['weather']]
 wx_codes = {'': 0, 'HZ': 5, 'BR': 10, '-DZ': 51, 'DZ': 53, '+DZ': 55,
             '-RA': 61, 'RA': 63, '+RA': 65, '-SN': 71, 'SN': 73, '+SN': 75}
-wx = [wx_codes[s.split()[0] if ' ' in s else s] for s in wx_text]
+wx = [wx_codes[s.split()[0] if ' ' in s else s] for s in data['weather'].fillna('')]
```

Comment:
Should the `fillna` be a separate step right after the data are read in? Might make sense to have all the data format munging in one spot.

Reply:
Well in this case it's only the NA's for the wx column that I want to replace, since abs that had no wx observation have '', which get replaced with NA.